benchmarks/data/codes/pt_bm.py

```diff
@@ -45,6 +45,8 @@
 Pd1s2 = g4 * dd_bias[4][None, :]
 Pd2s2 = g4 * dd_bias[5][None, :]
 Ps2s2 = g4 * dd_bias[6][None, :]
+Pd1d3 = g4 * dd_bias[8][None, :]
```

Comment:
I think we should be careful with notation. This is b3nl, which is not the classic b3 (which gets renormalized) and isn't the factor in front of delta^3.